assets/js/googlesitekit/datastore/forms/index.test.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * core/forms data store tests.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { createTestRegistry } from 'tests/js/utils';
+import { STORE_NAME } from './constants';
+
+describe( 'core/forms store', () => {
+	let registry;
+
+	beforeEach( () => {
+		registry = createTestRegistry();
+	} );
+
+	describe( 'actions', () => {
+		describe( 'setValues', () => {
+			it( 'requires the formName param', () => {
+				expect( () => {
+					registry.dispatch( STORE_NAME ).setValues();
+				} ).toThrow( 'formName is required for setting values.' );
+			} );
+
+			it( 'requires the formData param', () => {
+				expect( () => {
+					registry.dispatch( STORE_NAME ).setValues( 'form-name' );
+				} ).toThrow( 'formData must be an object.' );
+			} );
+
+			it( 'requires the formData param to be an object not an array', () => {
+				expect( () => {
+					registry.dispatch( STORE_NAME ).setValues( 'form-name', [] );
+				} ).toThrow( 'formData must be an object.' );
+			} );
+
+			it( 'requires the formData param to be an object not a string', () => {
```

Comment:
You could add another test here ensuring an object _doesn't_ throw, especially an empty one:

```js
it( 'does not throw if formData is an object', () => {
    expect( () => {
        registry.dispatch( STORE_NAME ).setValues( 'form-name', {} );
    } ).not.toThrow();
} );
```